projects/components/src/utils/test/widget-object/cypress/cypress-widget-finder.ts

```diff
@@ -40,11 +40,23 @@ export class CypressWidgetObjectFinder<T> {
         if (findOptions?.cssSelector) {
             tagName += `${findOptions.cssSelector}`;
         }
+
         const id = idGenerator.generate();
-        const search = findOptions?.ancestor
-            ? cy.get(findOptions.ancestor).find(tagName)
-            : cy.get(tagName, findOptions?.options);
-        const root = search.as(id);
+
+        const ancestor = findOptions?.ancestor ? cy.get(findOptions?.ancestor) : cy.get('body');
```

Comment:
We're not using the `FindCypressWidgetOptions.options` for timeout.

For example:
```
// This code isn't complete. Don't pass in `timeout` if it's not defined.
const timeout = findOptions?.options?.timeout;
const ancestor = findOptions?.ancestor ? cy.get(findOptions?.ancestor, { timeout } ) : cy.get('body',  { timeout });
```

Reply:
Updated to include this!